test/com/gfredericks/test/chuck/properties_test.cljc

```diff
@@ -0,0 +1,49 @@
+(ns com.gfredericks.test.chuck.properties-test
+  #?(:cljs (:require-macros [com.gfredericks.test.chuck.properties.macros :as prop']
+                            [com.gfredericks.test.chuck.generators :as gen']))
+  #?(:clj  (:require [clojure.test :refer :all]
+                     [clojure.test.check :as t.c]
+                     [clojure.test.check.generators :as gen]
+                     [clojure.test.check.clojure-test :refer [defspec]]
+                     [com.gfredericks.test.chuck.properties :as prop'])
+     :cljs (:require [cljs.test :refer-macros [run-tests deftest is]]
+                     [cljs.test.check :as t.c]
+                     [cljs.test.check.generators :as gen]
+                     [cljs.test.check.properties :include-macros true]
+                     [cljs.test.check.cljs-test :refer-macros [defspec]])))
```

Comment:
Thanks. I'm making progress in using the unified namespaces: generators-test is working, properties-test is closer to be working. I should be able to push something or report issues soon.